### PR TITLE
Add ufal.udpipe

### PR DESCRIPTION
**Pull request by ajdapretnar**
Recipe for the ufal.udpipe package.

### CODE REVIEW

File: recipes/ufal.udpipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "ufal.udpipe" %}
+{% set version = " 1.2.0.1 " %}
```
Comment (PrimozGodec) [inline recipes/ufal.udpipe/meta.yaml]: @ajdapretnar please remove spaces around version.

File: recipes/ufal.udpipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (jakirkham) [inline recipes/ufal.udpipe/meta.yaml]: Please add the following to the `build` section here.

```yaml
  skip: true  # [win and vc<14]
```

This should drop the Windows builds where there is not a new enough compiler.

Comment (ajdapretnar) [reply recipes/ufal.udpipe/meta.yaml]: Done. Thanks!

**Comment by ajdapretnar** [comment]
We urgently need this for the orange3-text add-on to start working. Would it be possible to merge soon, since it looks like tests now pass? Thanks!

**Comment by PrimozGodec** [comment]
@conda-forge/staged-recipes

File: recipes/ufal.udpipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    as a library for C++, Python, Perl, Java, C#, and as a web service. Third-party 
+    R CRAN package also exists.
+  doc_url: http://ufal.mff.cuni.cz/udpipe/users-manual
+  dev_url: https://github.com/ufal/udpipe
+
+extra:
+  recipe-maintainers:
+    - ajdapretnar
+    - primozgodec
+    - astaric
```
Comment (marcelotrevisani) [inline recipes/ufal.udpipe/meta.yaml]: All of these people already agreed to be a maintainer as well?

@primozgodec
@astaric

Comment (astaric) [reply recipes/ufal.udpipe/meta.yaml]: I am ok with being a maintainer

Comment (PrimozGodec) [reply recipes/ufal.udpipe/meta.yaml]: I agree

**Comment by marcelotrevisani** [comment]
LGTM!

Thanks to put this in ``conda-forge`` !  :)

**Pull request merged**